azurerm/resource_arm_service_fabric_cluster.go

```diff
@@ -53,6 +53,11 @@ func resourceArmServiceFabricCluster() *schema.Resource {
 				}, false),
 			},
 
+			"cluster_code_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
Could we get some validation here? Regex to enfore a `x.x.x.x` version would be nice but a `validation.NoZeroValues` is better then nothing